_posts/2013-05-02-install.markdown

```diff
@@ -219,16 +221,32 @@ Open [whatbrowser.org](http://whatbrowser.org) and update your browser if you do
 This is not strictly necessary, but it avoids a problem with and `ExecJS::RuntimeError` that might
 occur later  ([see stackoverflow](https://stackoverflow.com/questions/12520456/execjsruntimeerror-on-windows-trying-to-follow-rubytutorial)).
 
-* Go to [https://nodejs.org/](https://nodejs.org/) and install node
-* For windows users: reopen your Rails Command Shell
+* Go to [https://nodejs.org/](https://nodejs.org/) and install node LTS package
+* Reopen your Rails Command Shell
 
 Check your version of node
 
 {% highlight sh %}
 node --version
 {% endhighlight %}
 
-Make sure it is higher than `0.12`.
+Make sure it is displaying version number.
+
+### *5.* Check the environment
+
+Make sure that all works well by running the application generator command.
+
+{% highlight sh %}
+rails new myapp
+cd myapp
+rails server
+{% endhighlight %}
+
+Access to `http://localhost:3000` on browser, and you can see the 'Yay! You're on Rails!' page.
```

Comment:
"Go to `http://...` in your browser, and you should see the '....' page." is better I think.